hack/generate.sh

```diff
@@ -76,10 +76,6 @@ function generate_configuration() {
     docker run --rm -v ${__root}/config/onprem-iso-fcc.yaml:/config.fcc:z quay.io/coreos/fcct:release --pretty --strict /config.fcc > ${__root}/config/onprem-iso-config.ign
 }
 
-function generate_ocp_version() {
-  generate_configuration
-}
```

Comment:
Why was this removed?

Reply:
That was a temporary backwards-compatibility wrapper; it's no longer needed, as https://github.com/openshift/assisted-test-infra/commit/5ba6a083ee61e02afa70a30fd154be35700a82aa and https://github.com/openshift-assisted/assisted-installer-deployment/commit/9d34ed4518666b31784d6b85ece0838c71a0b341 were merged.

Reply:
Okay, but to be clear this doesn't have anything to do with the rest of the changes made in this PR, right?
I'm asking because it doesn't seem related to the env var change.

Reply:
It's not strictly related. It's a clean-up after some other env variables-related PR.